teamscale_client/client.py

```diff
@@ -680,6 +680,9 @@ def get_findings(self, uniform_path, timestamp, recursive=True):
             timestamp (datetime.datetime): timestamp (unix format) for which to upload the data
             recursive (bool): Whether to query findings recursively, i.e. also get findings for files under the given
                 path.
+            filter: Whether to filter specific findings from the result
```

Comment:
Document that this can either be a single filter, i.e. "category/group" string or a list of these strings to filter for multiple groups?

Reply:
I have pulled the same notes that the REST API documentation has on these parameters.